jobs/JGLOBAL_PREP_OCEAN_OBS

```diff
@@ -1,21 +1,29 @@
 #!/bin/bash
 source "${HOMEgfs}/ush/preamble.sh"
-source "${HOMEgfs}/ush/jjob_header.sh" -e "prepoceanobs" -c "base prepoceanobs"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "prepoceanobs" -c "base ocnanal prepoceanobs"
```

Comment:
this script should not depend on `config.ocnanal`.

Reply:
if it does, we need to understand why.  
The j-jobs in global-workflow typically source `config.base` and `config.job`.  There are some exceptions however, e.g. `efcs` sources `fcst` to get model configuration for deterministic and over-ride for the ensemble and `eobs` sources `anal` because it runs the GSI as an observer.

Reply:
@aerorahul All that is needed from `ocnanal` (I think) is the environment variable that points to the `obs_list.yaml`. The list is used to figure out what to copy from the 6 hr tank. 
@AndrewEichmann-NOAA , can you add that variable to `config.prepoceanobs` and remove `ocnanal`?

Reply:
The tasks `ocnanal` and `prepoceanobs` have certain environment variables that should be identical. Should the assignments be duplicated in `config. prepoceanobs`?

Reply:
@guillaumevernieres Working on it (and testing)

Reply:
@aerorahul Both tasks share the environment variable SOCA_OBS_LIST, which should point to the same file. Should it be set in `config.prepoceanobs` as well, in order to not run `config.ocnanal`?

Reply:
I'm of the opinion that these things (`OBS_LIST`, etc.) should either be in config.FOOanl, or a new config.FOOobs that is sourced by all of the analysis/prep jobs for FOO (atm, ocn, aero, land)

Reply:
We can copy but we can't move anything yet @aerorahul . We're hopping to change this soon, but the current obs are still fetched with r2d2, under the marine DA prep task.

Reply:
@guillaumevernieres Do you have any preference on how to handle the config files?